interactive-etl/setup.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+set -u 
+set -e
+set -o errexit
+
+NAMESPACE=${1:-flink}
+
+printf "\n\nInstalling example components into namespace: $NAMESPACE\n\n"
+
+# Install CertManager - this is needed by the Flink Kubernetes Operator
+printf "\nChecking for CertManager install\n"
+if kubectl get namespace cert-manager ; then
+    printf "CertManager is already installed\n"
+else
+    kubectl create -f https://github.com/jetstack/cert-manager/releases/download/v1.8.2/cert-manager.yaml
+fi
+
+# Add the Flink operator's helm repo
+helm repo add flink-operator-repo https://downloads.apache.org/flink/flink-kubernetes-operator-1.9.0/
+
+printf "\nChecking for $NAMESPACE namespace\n"
+if kubectl get namespace $NAMESPACE ; then
+    echo "$NAMESPACE namespace already exists"
+else
+    kubectl create namespace $NAMESPACE
+fi
+
+printf "\nWaiting for cert-manager webhook to be ready..."
+kubectl -n cert-manager wait --for=condition=Available --timeout=60s deployment cert-manager-webhook
+
```

Comment:
From my experience this is still not enough for be sure that cert-manager is completely up and running and prepared for serving certs. In test suite we also added additional 1m timeout to not hit issues with flink operator install.